tests/specs/config.ts

```diff
@@ -47,8 +69,8 @@ export default testSuite(({ describe }) => {
 
 			expect(stdout).toBe('');
 			expect(stderr).toBe('');
-		});
 
-		await fixture.rm();
+			await fixture.rm();
+		});
```

Comment:
I considered refactoring this file to run independently but these all finish instantly so I personally didn't see any benefit.

Also, even though there's multiple `test()` calls, since they all just test the config command, I look at it as one big test with sub-labels.

Reply:
I don't think having test run independently should be about performance or test speed. I think it should always be about the benefits of having a test in isolation:. https://www.telerik.com/blogs/why-isolating-your-unit-tests-matters.

When a test start failing you can pinpoint it to the exact test that causes it to fail and not have to think about the complexity of multiple tests influencing a test. Easier debugging, etc.

Although this test file is small and there is not that much to it, I think that this rule should be applied also for the sake of consistency :).